docs/editors/emacs.md

```diff
@@ -73,7 +73,8 @@ To use Metals in Emacs, place this snippet in your Emacs configuration (for exam
   ;; (setq lsp-idle-delay 0.500)
   ;; (setq lsp-log-io nil)
   ;; (setq lsp-completion-provider :capf)
-  (setq lsp-prefer-flymake nil))
+  (setq lsp-prefer-flymake nil)
+  (setq lsp-keep-workspace-alive nil))
```

Comment:
Perhaps add a comment as to what this does?

Reply:
Done!